specification/types.md

```diff
@@ -125,7 +125,7 @@ A structure defining a provider event payload, including:
 
 A structure defining an event payload, including:
 
-- client name (string, required)
```

Comment:
Perhaps you could add this as optional since it's already available in some SDKs.

Reply:
When we say something is "optional" in all other places it means the SDK must define it, but it can be unset. We could add a note or sub-point about it, but TBH I'd rather just leave it unspecified. I think it offers much less value than the provider name.